cloud_build/wsi2dcm.rb

```diff
@@ -0,0 +1,32 @@
+class Wsi2dcm < Formula
+  desc "Conversion tool/library for converting whole slide images to DICOM "
+  version "0.0.1"
+  homepage "https://github.com/GoogleCloudPlatform/wsi-to-dicom-converter"
+  url "https://github.com/pavertomato/wsi-to-dicom-converter-1/archive/reviewBuild.zip"
```

Comment:
these two lines needs to be replaced as new release with macos fixes will be avalible on https://github.com/GoogleCloudPlatform/wsi-to-dicom-converter/releases page

Reply:
Can we put the homebrew part here in it's own PR, so that we can submit the MacOS fixes separately and do a release before adding this?

Reply:
reverted, wsi2dcm.rb will wait until release